src/dvclive/live.py

```diff
@@ -449,6 +446,19 @@ def log_artifact(
                     )
 
     def cache(self, path):
+        if self._inside_dvc_exp:
```

Comment:
I would prefer to include and review this in a separate P.R.

Reply:
I moved the other parts to #632, so PTAL and let's keep this PR open to discuss the `_inside_dvc_exp` logic.